### ISSUE TITLE
Create a jupyter-slides (name to be discussed ;-) organization to house several Jupyter slideshows projects

### ISSUE THREAD

**Issue by damianavila** [damianavila/RISE#635]
@bollwyvl captured quite well the story around presenting with Jupyter at https://github.com/deathbeds/jupyterlab-deck/blob/main/examples/History.ipynb.

Currently, we have RISE (supporting the v6 notebook), the JLab-compatible incantation that was pushed forward by @fcollonval as a PR on this very same repo (https://github.com/damianavila/RISE/pull/605) but eventually landed in https://github.com/jupyterlab-contrib/rise, and a new attempt to solve the problem called jupyterlab-deck living at https://github.com/deathbeds/jupyterlab-deck.

I think we can all benefit if we join forces with a GH organization that can sustain independent but super-related projects like the ones I mentioned above. For instance, the JLab-compatible rise implementation now lives in a different repo (for simplicity at the time to generate a package) but it obviously shares a lot of stuff with the original v6-compatible RISE. The jupyterlab-deck project is using the already established nbconvert/RISE metadata, so any changes in that aspect of RISE going forward can be (previously to be implemented) discussed and agreed on so jupyterlab-deck keeps compatibility on that aspect (or maybe influence RISE somehow in the other direction!!).

I also see how other implementations of the "Jupyter slides" (presenting with Jupyter) can converge on this yet-to-be-created new organization, giving people enough freedom to keep exploring the space on their own independent repo but benefiting at the same time to have a shared space where we can all discuss ideas relevant for this topic.

@bollwyvl, @fcollonval, @parmentelat, WDYT?

In summary, the proposal follows:

1. [ ] Create a new GH organization (Jupyter Slides... suggestions for other names are welcome :wink:).
2. [ ] Move repos into the new org:
  * [ ] https://github.com/damianavila/RISE
  * [ ] https://github.com/jupyterlab-contrib/rise
  * [ ] https://github.com/deathbeds/jupyterlab-deck
  * [ ] other repos??
3. [ ] Set up permissions properly so everyone has the flexibility they want on those repos.
4. [ ] Tell people about this!!
5. [ ] Do more fun stuff!!

Btw, I will be more than happy to push forward on the steps I am proposing above if we get a general agreement.

Thoughts are welcome!!

**Comment by fcollonval** [comment]
Thanks @damianavila 

Yes it sounds like a good plan.

**Comment by parmentelat** [comment]
sounds like a great idea indeed !

**Comment by westurner** [comment]
- https://github.com/jupyter
  - https://github.com/jupyter/governance
  - https://github.com/jupyterhub
  - https://github.com/jupyterlab
- https://github.com/jupyter-contrib

Proposed;

- https://github.com/jupyter-slides

There are many great slides solutions. IDK which org repo is best or necessary?

It would be great to get something merged into JupyterLab (and thereby also Retro Notebook, which is based on JupyterLab instead of nbclassic) that's better than Reveal.js and "Presentation Mode".

**Comment by westurner** [comment]
> * https://github.com/jupyter
>   
>   * https://github.com/jupyter/governance

https://jupyter.org/governance/overview.html#other-major-components-of-the-organization :

- Distinguished Contributors
- Standing Committees and Working Groups
- Software Subprojects
 
https://jupyter.org/governance/software_subprojects.html#responsibilities-of-jupyter-subprojects

**Comment by westurner** [comment]
- DOC: Recommend testing Jupyter slide extensions and native functionality w/ JupyterLite, or is that necessary? 
  - https://github.com/executablebooks/jupyter-book/issues/1661#issuecomment-1066935363

**Comment by westurner** [comment]
> https://jupyter.org/governance/software_subprojects.html#responsibilities-of-jupyter-subprojects
> 
> https://jupyter.org/governance/list_of_subprojects.html#list-of-official-jupyter-subprojects :
> 
> > * Official Subprojects with SSC representation
> > * Official Subprojects without SSC representation

- https://github.com/jupyter/governance/pull/161

**Comment by cpjobling** [comment]
As a long-term user of RISE in my teaching and an early adopter of the [jupyterlab-myst](https://github.com/executablebooks/jupyterlab-myst) extension (which needs JL 4) I would certainly be interested in this project. 

Currently, jupyterlab-deck is the only extension I've found that supports the enhanced outputs provided by the myst extension, but its navigation doesn't seem to work properly: particularly for subslides. 

The jupyterlab-rise extension exists, but it is missing some of the UI elements that RISE provides. In particular, the facility to show the slide controls in the cells. There is no usage documentation, so it is unclear how RISE is supposed to work in Jupyter Lab. Furthermore, any myst blocks that are rendered in the jupyter lab editor, are not rendered in the slideshow. (This is also true of `nbconvert --to slides` so the solution may require the cooperation of the jupyterlab-myst project.)

I'm happy to help in testing.
